app/views/api/v1/topics/targets/destroy.json.jbuilder

```diff
@@ -0,0 +1 @@
+json.partial! 'api/v1/targets/info', target: target
```

Comment:
we do not need a view for destroy.. we just return `head :no_content` and that's enough

Reply:
this was fixed in the last commit